src/libraries/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Android;$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
```

Comment:
Not sure if this is the right `csproj` file and all of these are needed here. This was copy-paste to get it working.

Reply:
I do not think you need `iOS` and `tvOS`. The implementation does appear to be special cased for them, so they can just inherit the `-Unix` implementation.